browser/resources/settings/brave_content_page/containers.css

```
@@ -42,3 +42,52 @@
 .container:last-of-type {
   border-bottom: none;
 }
+
+.container .icon-and-label {
+  display: flex;
+  align-items: center;
+  max-width: calc(100% - 90px);
+  gap: var(--leo-spacing-xl);
+  font:var(--leo-font-default-regular);
+}
+
+.container .icon-and-label settings-brave-content-containers-icon {
+  width: 20px;
+  height: 20px;
+  --leo-icon-size: 16px;
+  flex-shrink: 0;
+}
```

nit: variables should be first
```suggestion
.container .icon-and-label settings-brave-content-containers-icon {
  --leo-icon-size: 16px;

  width: 20px;
  height: 20px;
  flex-shrink: 0;
}
```